cmd/meroxa/root/apps/init.go

```diff
@@ -64,12 +65,49 @@ func (i *Init) ParseArgs(args []string) error {
 }
 
 func (i *Init) GitInit(ctx context.Context, path string) error {
+	// should this be in turbine?
```

Comment:
@janelletavares I think this could be probably be moved to `turbine-cli` in anticipation to, maybe, eventually move it to its own CLI. Let's not worry about that for now if you think that adds scope.